{
	"id": "oval:org.almalinux.alsa:def:20240116",
	"version": "638",
	"class": "patch",
	"metadata": {
		"Title": "ALSA-2024:0116: python-urllib3 security update (Moderate)",
		"reference": [
			{
				"ref_id": "RHSA-2024:0116",
				"source": "RHSA",
				"ref_url": "https://access.redhat.com/errata/RHSA-2024:0116"
			},
			{
				"ref_id": "CVE-2023-43804",
				"source": "CVE",
				"ref_url": "https://access.redhat.com/security/cve/CVE-2023-43804"
			},
			{
				"ref_id": "CVE-2023-45803",
				"source": "CVE",
				"ref_url": "https://access.redhat.com/security/cve/CVE-2023-45803"
			},
			{
				"ref_id": "ALSA-2024:0116",
				"source": "ALSA",
				"ref_url": "https://errata.almalinux.org/8/ALSA-2024-0116.html"
			}
		],
		"Description": "The python-urllib3 package provides the Python HTTP module with connection pooling and file POST abilities.\n\nSecurity Fix(es):\n\n* python-urllib3: Cookie request header isn't stripped during cross-origin redirects (CVE-2023-43804)\n* urllib3: Request body not stripped after redirect from 303 status changes request method to GET (CVE-2023-45803)\n\nFor more details about the security issue(s), including the impact, a CVSS score, acknowledgments, and other related information, refer to the CVE page(s) listed in the References section.",
		"advisory": {
			"from": "packager@almalinux.org",
			"Severity": "Moderate",
			"Rights": "Copyright 2024 AlmaLinux OS",
			"issued": {
				"date": "2024-01-10"
			},
			"updated": {
				"date": "2024-01-10"
			},
			"affected_cpe_list": {
				"Cpe": [
					"cpe:/a:almalinux:almalinux:8",
					"cpe:/a:almalinux:almalinux:8::appstream",
					"cpe:/a:almalinux:almalinux:8::powertools",
					"cpe:/a:almalinux:almalinux:8::highavailability",
					"cpe:/a:almalinux:almalinux:8::nfv",
					"cpe:/a:almalinux:almalinux:8::realtime",
					"cpe:/a:almalinux:almalinux:8::resilientstorage",
					"cpe:/a:almalinux:almalinux:8::sap",
					"cpe:/a:almalinux:almalinux:8::sap_hana",
					"cpe:/a:almalinux:almalinux:8::supplementary",
					"cpe:/o:almalinux:almalinux:8",
					"cpe:/o:almalinux:almalinux:8::baseos"
				]
			},
			"bugzilla": [
				{
					"href": "https://bugzilla.redhat.com/2242493",
					"id": "2242493"
				},
				{
					"href": "https://bugzilla.redhat.com/2246840",
					"id": "2246840"
				}
			],
			"cve": [
				{
					"text": "CVE-2023-43804",
					"public": "20231004",
					"href": "https://access.redhat.com/security/cve/CVE-2023-43804",
					"impact": "Moderate",
					"cwe": "CWE-200",
					"cvss3": "CVSS:3.1/AV:N/AC:H/PR:H/UI:N/S:U/C:H/I:H/A:N"
				},
				{
					"text": "CVE-2023-45803",
					"public": "20231013",
					"href": "https://access.redhat.com/security/cve/CVE-2023-45803",
					"impact": "Moderate",
					"cwe": "CWE-200",
					"cvss3": "CVSS:3.1/AV:A/AC:H/PR:H/UI:N/S:U/C:H/I:N/A:N"
				}
			]
		}
	},
	"criteria": {
		"operator": "OR",
		"criterias": [
			{
				"operator": "AND",
				"criterias": [
					{
						"operator": "OR",
						"criterions": [
							{
								"test_ref": "oval:org.almalinux.alsa:tst:2020034807012",
								"comment": "python3-urllib3 is signed with AlmaLinux OS 8 key (51d6647ec21ad6ea)"
							},
							{
								"test_ref": "oval:org.almalinux.alsa:tst:2020034807013",
								"comment": "python3-urllib3 is signed with AlmaLinux OS 8 key (2ae81e8aced7258b)"
							}
						]
					}
				],
				"criterions": [
					{
						"test_ref": "oval:org.almalinux.alba:tst:20191992003",
						"comment": "AlmaLinux 8 is installed"
					},
					{
						"test_ref": "oval:org.almalinux.alsa:tst:20240116001",
						"comment": "python3-urllib3 is earlier than 0:1.24.2-5.el8_9.2"
					}
				]
			}
		],
		"criterions": [
			{
				"test_ref": "oval:org.almalinux.alba:tst:20191992005",
				"comment": "AlmaLinux must be installed"
			}
		]
	}
}
